testsuite/tests/backtrace/lazy.ml

```diff
@@ -1,6 +1,12 @@
 (* TEST
-   flags = "-g"
-   * native
+
+flags = "-g"
+* no-flambda
+** native
+* flambda
+reference = "${test_source_directory}/lazy.flambda.reference"
+** native
+
```

Comment:
Could you use TEST_BELOW to avoid line number updates in the future? See tests/backtrace/backtrace.ml.